doc/radosgw/account.rst

```diff
@@ -187,6 +187,43 @@ Alternatively, you may want to create a new account for each existing user. In
 that case, you may want to add the ``--account-root`` option to make each user
 the root user of their account.
 
+Migrating Notification Topics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Account topics are only supported when ``notification_v2`` feature is enabled,
+as described in `Bucket Notifications`_ and `Supported Zone Features`_.
+
+1. ``Migration Impact``: When a non-account user is migrated to an account,
+the user loses access to its existing notification topics via the SNS Topic API.
+However, ongoing notifications remain unaffected.
+
+2. ``Recreation of Topics``: The account-user should recreate the topics using
```

Comment:
I personally write `re-create` when not discussing fun and games, though this is somewhat controversial.

I would take out the hypen between `account` and `user.

Reply:
Done.